### ISSUE TITLE
error with new achievements

### ISSUE THREAD

**Issue by Lacro59** [felixkmh/StartPage-for-Playnite#63]
I'd recently Genshin Impact achievements and your plugin crash Playnite because it don't find the local image.
When I disabled your plugin, I've not any crash.

```28-01 17:26:02.927|ERROR|PlayniteApplication:Unhandled exception occured.
System.Windows.Markup.XamlParseException: L'initialisation de 'System.Windows.Media.Imaging.BitmapImage' a levé une exception. ---> System.IO.IOException: Impossible de trouver la ressource 'views/genshinimpact/ac.png'.
   à MS.Internal.AppModel.ResourcePart.GetStreamCore(FileMode mode, FileAccess access)
   à System.IO.Packaging.PackagePart.GetStream(FileMode mode, FileAccess access)
   à System.IO.Packaging.PackWebResponse.CachedResponse.GetResponseStream()
   à System.IO.Packaging.PackWebResponse.GetResponseStream()
   à System.IO.Packaging.PackWebResponse.get_ContentType()
   à System.Windows.Media.Imaging.BitmapDecoder.SetupDecoderFromUriOrStream(Uri uri, Stream stream, BitmapCacheOption cacheOption, Guid& clsId, Boolean& isOriginalWritable, Stream& uriStream, UnmanagedMemoryStream& unmanagedMemoryStream, SafeFileHandle& safeFilehandle)
   à System.Windows.Media.Imaging.BitmapDecoder.CreateFromUriOrStream(Uri baseUri, Uri uri, Stream stream, BitmapCreateOptions createOptions, BitmapCacheOption cacheOption, RequestCachePolicy uriCachePolicy, Boolean insertInDecoderCache)
   à System.Windows.Media.Imaging.BitmapImage.FinalizeCreation()
   à System.Windows.Media.Imaging.BitmapImage.EndInit()
   à MS.Internal.Xaml.Runtime.ClrObjectRuntime.InitializationGuard(XamlType xamlType, Object obj, Boolean begin)```

**Comment by felixkmh** [comment]
Weird, it should fall back to using the url as the image source if the cached image does not exist. Does the extensions.log contain any logs from "LandingPage" or "StartPage"?

**Comment by felixkmh** [comment]
Or could you upload the successstory json file associated with Genshin Impact? I don't think it is crashing when trying to load the local file (this part is in a try-catch block), so it must be crashing when trying to load the file from the url inside the UrlUnlocked property.

**Comment by Lacro59** [comment]
The url of achievement image is a relataive path because the ressource in plugin folder in ressource directory.

**Comment by felixkmh** [comment]
fixed via c2ddde073c1d3b71b9c85d4282d23ded62caa86c
I'm just going to ignore uris that aren't web links
